test/Build.skopeo-umoci

```diff
@@ -1,37 +0,0 @@
-#!/bin/bash
```

Comment:
Why delete this file?

Reply:
I removed it because the functionality it tested is now included in this script and tested by the  new ch-pull2tar test.

Reply:
Sounds good.

Reply:
Sounds good!